ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -53,6 +53,27 @@ class DemoListViewController: UITableViewController {
         tableView.register(TableViewCell.self, forCellReuseIdentifier: cellReuseIdentifier)
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if didAutoLaunch {
+            UserDefaults.standard.set(nil, forKey: DemoListViewController.lastDemoControllerKey)
+        } else {
+            let lastDemoController = UserDefaults.standard.string(forKey: DemoListViewController.lastDemoControllerKey)
+
+            if lastDemoController?.count ?? -1 > 0 {
```

Comment:
How about this logic to find the index? (considering the isFirstLaunch variable name)

        if !isFirstLaunch {
            UserDefaults.standard.set(nil, forKey: DemoListViewController.lastDemoControllerKey)
        } else {
            isFirstLaunch = false

            if let lastDemoController = UserDefaults.standard.string(forKey: DemoListViewController.lastDemoControllerKey),
               let index = demos.firstIndex(where: { $0.title == lastDemoController }) {
                tableView(tableView, didSelectRowAt: IndexPath(row: index, section: 0))
            }
        }